plugins/obs-ffmpeg/jim-nvenc.c

```diff
@@ -922,6 +922,17 @@ static bool init_encoder_av1(struct nvenc_data *enc, obs_data_t *settings,
 	if (config->rcParams.rateControlMode == NV_ENC_PARAMS_RC_CBR)
 		av1_config->enableBitstreamPadding = 1;
 
+#define FOUR_K_CX 3840
+#define FOUR_K_CY 2160
+#define PIXELCOUNT_4K (FOUR_K_CX * FOUR_K_CY)
+
+	/* If 4K+, set tiles to 2x2 uniform tiles. */
+	if ((voi->width * voi->height) >= PIXELCOUNT_4K) {
+		av1_config->enableCustomTileConfig = 0;
```

Comment:
Any reason the NVENC code uses macros and the AMD code uses `constexpr`? I'd prefer they are consistent over the PR.

Reply:
C doesn't have constexpr, otherwise I'd use it. They're functionally identical otherwise.

Reply:
To answer your question, it's cleaner, and constexpr doesn't leave scope. Acts like a normal constant but is instead a compile-time constant

Reply:
Ah, I missed that. I suppose we could use `const int` instead and let compiler optimizations take care of it for a similar result.

Reply:
As a note for the future, C23 introduces constexpr, and constexpr ints should work once we support C23. Until then, as mentioned above, C does not have constexpr.